docs/data/system/getting-started/usage/usage.md

```diff
@@ -366,6 +366,53 @@ declare module '@mui/material/styles' {
 }
 ```
 
+#### Container queries
+
+In order to use [`@container` queries](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Container_Queries) instead of [`@media` queries](https://developer.mozilla.org/en-US/docs/Web/CSS/Media_Queries), you can add the prefix `cq` to your sx definitions.
+
+{{"demo": "ContainerBreakpointsAsObject.js"}}
+
+#### Container query breakpoints
+
+In order to use different breakpoints for container queries than the default `xs` to `xl`, you may define them with the same prefix as in the sx props.
+
+```jsx
+import * as React from 'react';
+import Box from '@mui/material/Box';
+import { createTheme, ThemeProvider } from '@mui/material/styles';
+
+const theme = createTheme({
```

Comment:
```suggestion
const themeWithCustomBreakpoints = createTheme({
```

For some clarity. Otherwise, I feel that the use of "custom breakpoints" text in the following code can possibly confuse the reader

Reply:
s